javascript/javascript3/week3/art/art.js

```diff
@@ -0,0 +1,64 @@
+// Paint a circle to a canvas element
+const canvas = document.getElementById('canvas');
+const context = canvas.getContext('2d');
+
+context.fillStyle = 'red';
+context.beginPath()
+context.arc(50, 50, 50, 0, 2 * Math.PI);
+context.stroke();
+context.fill();
+
+
+
+// Class creation time!
+class Circle {
+    constructor(x, y, r, startAngle, endAngle, fillColor) {
+        this.x = x
+        this.y = y
+        this.r = r
+        this.startAngle = startAngle
+        this.endAngle = endAngle
+        this.fillColor = fillColor
+    }
+    draw() {
+        context.beginPath()
+        context.fillStyle = this.fillColor;
+        context.arc(this.x, this.y, this.r, this.startAngle, this.endAngle, this.fillColor)
+        context.stroke()
+        context.fill();
+    }
+}
+const c1 = new Circle(175, 175, 20, 0, 2 * Math.PI, '#000000');
+const c2 = new Circle(130, 100, 40, 0, 2 * Math.PI, 'blue');
+c1.draw();
+c2.draw();
+
+
+canvas.setAttribute('width', window.innerWidth);
+canvas.setAttribute('height', window.innerHeight);
+
+
+
+//  Now lets make art!
+function getRandomCircles() {
```

Comment:
get () implies you are getting some value, and NOT drawing anything. The drawing part should be somewhere else.